### PR TITLE
Remove gradient from the cone as as it's duplicate information

### PR DESCRIPTION
**Pull request by TrentHouliston**
Radius and gradient are the same number and having both is terrible

### CODE REVIEW

File: module/vision/BallDetector/src/BallDetector.cpp [hunk 1]
```diff
@@ -147,10 +147,9 @@ namespace vision {
                         }
 
                         // Ball cam space info
-                        b.cone.axis     = horizon.Hcw.topLeftCorner<3, 3>().cast<float>() * axis;
-                        float proj      = 1.0f / radius;
-                        b.cone.gradient = std::sqrt(proj * proj - 1.0f);
-                        b.cone.radius   = radius;
+                        b.cone.axis   = horizon.Hcw.topLeftCorner<3, 3>().cast<float>() * axis;
+                        float proj    = 1.0f / radius;
```
Comment (Bidski) [inline module/vision/BallDetector/src/BallDetector.cpp:151]: Is `proj` still used?

File: shared/message/Geometry.proto [hunk 2]
```diff
@@ -58,7 +58,7 @@ message Frustum {
 }
 
 message Cone {
-    fvec3 axis     = 1;
-    float gradient = 2;
-    float radius   = 3;
+    fvec3 axis = 1;
+    // Cos(theta) of the cone angle
+    float radius = 3;
```
Comment (Bidski) [inline shared/message/Geometry.proto:63]: Should we update the index?

Comment (TrentHouliston) [reply shared/message/Geometry.proto:63]: Would mean old NBS files won't play properly... Do we have any we care about?

File: module/vision/BallDetector/src/BallDetector.cpp (hunk 1 above)
Comment (TrentHouliston) [reply module/vision/BallDetector/src/BallDetector.cpp]: Removed

File: shared/message/Geometry.proto (hunk 2 above)
Comment (Bidski) [reply shared/message/Geometry.proto:63]: good point, we probably do have some

**Pull request merged**